prow/plugins/trigger/push.go

```diff
@@ -17,12 +17,31 @@ limitations under the License.
 package trigger
 
 import (
+	"fmt"
+
 	"k8s.io/test-infra/prow/github"
 	"k8s.io/test-infra/prow/kube"
 	"k8s.io/test-infra/prow/plank"
 )
 
+func notifyOnSlackIfManualMerge(sc slackClient, pe github.PushEvent) error {
+	if sc == nil {
+		return nil
+	}
+	if pe.Pusher.Name != "k8s-merge-robot" && pe.Sender.Login != "k8s-merge-robot" {
```

Comment:
Please move this into a config option in `prow/config/config.go`. See the trigger options for an example of how to do so.